brain/python/compartmentReport.cpp

```diff
@@ -90,6 +90,10 @@ bp::object CompartmentReport_getMetaData(const CompartmentReport& reader)
     dict["time_step"] = md.timeStep;
     dict["time_unit"] = md.timeUnit;
     dict["data_unit"] = md.dataUnit;
+    dict["num_cells"] = md.cellCount;
+    dict["num_compartments"] = md.compartmentCount;
+    dict["num_frames"] = md.frameCount;
```

Comment:
In C++, the convention is to use count.
But the request was "num_*". May be a python convention. I'm not sure